src/commands/protocols/get/sender/v1.0.0/v1-0-0-get-request-command.js

```diff
@@ -31,14 +31,13 @@ class GetRequestCommand extends ProtocolRequestCommand {
     }
 
     async prepareMessage(command) {
-        const { assertionId, blockchain, contract, tokenId, hashFunctionId, state } = command.data;
+        const { blockchain, contract, tokenId, assertionId, state } = command.data;
 
         return {
-            assertionId,
             blockchain,
             contract,
             tokenId,
-            hashFunctionId,
```

Comment:
if this is not needed, can you also remove it from the handler in rpc controller ?